README.md

```diff
@@ -186,6 +187,33 @@ In other words, if the endpoint is a _qcow2_ file, or any file wrapped by tar (e
 
 An additional caveat is that currently the size is under reported for simple iso files, meaning a structured iso file that is not archived and/or compressed.
 
+### Start Cloning Images
+
+Verify the following file from the [example manifests](./manifests/example) exists:
```

Comment:
If the target pvc manifest does not exist what is the user supposed to do?

Reply:
changed.